woocommerce/woocommerce
# Update WC Admin Homescreen header icons

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #46234.

This PR updates the header icons on the WooCommerce Admin Homescreen. We decided to only update the homescreen header icons for now, as the other icons will be updated in the future.

 The changes are as follows:

- "Preview store" becomes a text link
- Update the icons for "display options", and "help".
- Existing logic for when to display these icons remains in place.

<img width="996" alt="Screenshot 2024-04-09 at 12 05 22" src="https://github.com/woocommerce/woocommerce/assets/4344253/d5a14f5b-a309-42b3-b039-07029b48e905">

<img width="998" alt="Screenshot 2024-04-09 at 12 05 31" src="https://github.com/woocommerce/woocommerce/assets/4344253/8ddf0dc2-346f-4e8f-a226-950744dc274e">



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `WooCommerce > Home`.
2. Observe the header icons match the new designs.
3. Go to `WooCommerce > Orders`.
4. Observe the header icons remain the same as before.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Update WC Admin Homescreen header icons

#### Comment 

</details>
